### PR TITLE
passkey: enabled benchmark

### PR DESCRIPTION
**Pull request by aramikm**
# Goal
The goal of this PR is to enable benchmark side of the passkey pallet

Related to #2197

# Checklist
- [x] Spec version incremented?

### CODE REVIEW

File: runtime/frequency/src/lib.rs [hunk 1]
```diff
@@ -1283,8 +1283,7 @@ mod benches {
 		[pallet_time_release, TimeRelease]
 		[pallet_capacity, Capacity]
 		[pallet_frequency_tx_payment, FrequencyTxPayment]
-		// Todo: uncomment after removing the feature flag
-		// [pallet_passkey, Passkey]
+		[pallet_passkey, Passkey]
```
Comment (wilwade) [inline runtime/frequency/src/lib.rs:1286]: Do we also need update the `pallets/passkey/src/weights.rs` file for this as well (beyond what happens at release time?)

Comment (aramikm) [reply runtime/frequency/src/lib.rs:1286]: Yes, I'm planning to run it right now.

**Review by wilwade — APPROVED** [review]
🚢 it!

File: pallets/passkey/src/weights.rs [hunk 2]
```diff
@@ -48,10 +48,10 @@ impl<T: frame_system::Config> WeightInfo for SubstrateWeight<T> {
...
 	fn validate() -> Weight {
 		// Proof Size summary in bytes:
-		//  Measured:  `235`
-		//  Estimated: `5078`
-		// Minimum execution time: 1_728_647_000 picoseconds.
-		Weight::from_parts(1_741_351_000, 5078)
+		//  Measured:  `168`
+		//  Estimated: `4088`
+		// Minimum execution time: 2_182_480_000 picoseconds.
+		Weight::from_parts(2_190_791_000, 4088)
```
Comment (aramikm) [inline pallets/passkey/src/weights.rs:54]: Increased a fair amount!

**Review by saraswatpuneet — APPROVED** [review]
Could have been cheaper 🚢

**Review by JoeCap08055 — APPROVED** [review]

**Review by enddynayn — APPROVED** [review]
👍 great!

**Pull request merged**